urbanairship/push/payload.py

```diff
@@ -145,7 +150,7 @@ def ios(alert=None, badge=None, sound=None, content_available=False,
     if media_attachment is not None:
         payload['media_attachment'] = media_attachment
     if priority is not None:
-        if priority not in {10, 5}:
+        if priority not in set((10, 5)):
```

Comment:
Oh my; I'm so surprised that 2.6 is even still an option these days -- do people really still use that?  Python 2.7 was released over seven years ago!